README.md

```diff
@@ -161,3 +166,14 @@ It can be invoked with:
 This project makes use of the semi-standard Rust [log crate](https://crates.io/crates/log) to provide runtime logging.
 In order to see the log, you can set the `RUST_LOG` environment value to a [standard log level value](https://docs.rs/log/0.4.14/log/enum.Level.html).
 Notably, our HTTP client library will emit a lot of helpful information about the request & response cycle at the _trace_ level.
+
+### Testing Artifact Generation
+
+Generation of build artifacts is done using a GitHub Actions workflow and in many cases cannot be done in a local
+development environment.  To test changes to the artifact output, you can follow this workflow:
```

Comment:
I'm not sure that's entirely accurate. Every aspect of the build artifact generation is doable locally and everything you see there was tested locally first. The actual release management, however, is not runnable locally.

Reply:
Ideally the build commands locally and in CI are the same, hence "make rpm" works locally and also works in CI.  In order to validate the CI workflow locally I need to recreate the commands one at a time, manually, so it is not locally repeatable.  Perhaps a "make deb" that works both locally and in CI would be useful, but then it should probably build inside a debian docker container so the environment is standardized no matter what host you want to build on.